addon/components/bs-accordion/item/title.hbs

```diff
@@ -18,12 +19,17 @@
 {{else}}
   <div
     class="card-header"
-    role="tab"
     ...attributes
-    {{on "click" this.handleClick}}
```

Comment:
Moving the event listener from `.card-header` to `.card-header h5 button` could be seen as a breaking change. Due to margin some parts of the header may not be interactive anymore. I would consider it a bug fix. But wanted to raise that concern anyways. I guess it's most likely causing issues with tests using wrong selectors.

This only affects Bootstrap v4.

Reply:
Yeah, this was probably done this way to make the whole header clickable.

For BS4 this will change that:
<img width="814" alt="image" src="https://user-images.githubusercontent.com/1325249/212168726-77f33c99-83e9-4474-bea5-c694947d0663.png">

But their example also has only the button clickable, so I guess it's ok then? Even more so since we are cutting a major release anyway, so we could even mark this as a breaking change!?